lib/iris/coords.py

```diff
@@ -1578,6 +1578,28 @@ def bounds(self, bounds):
         self._bounds = bounds
 
 
+class CellMeasures(AuxCoord):
```

Comment:
Our initial reasoning was that the cell measures attribute should behave in a similar way to an `AuxCoord` (data payload with associated metadata) and we knew it would be quick to implement. I think you're right, on closer inspection, that it could just as well be a subclass of `Coord` - it probably makes more sense to do that.

We did have a discussion about whether it should even be a coordinate object at all, with it being an attribute that has associated data rather than a coordinate describing any data dimension. What are your opinions on that?